modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/DataRegionMetricsImpl.java

```diff
@@ -30,7 +31,7 @@
  */
 public class DataRegionMetricsImpl implements DataRegionMetrics, AllocatedPageTracker {
     /** */
-    private final IgniteOutClosure<Float> fillFactorProvider;
+    private final IgniteOutClosure<T2<Long, Long>> fillFactorProvider;
```

Comment:
How often fill factor provider will be called? Would it create GC pressure to call it?

Reply:
@dspavlov I expect it will be called once per metrics beat, that means once per every 2 seconds. I don't think it will create much of a GC pressure.